rotkehlchen/data_import/importers/binance.py

```diff
@@ -338,7 +352,81 @@ def process_entry(
             link=None,
             notes=f'Imported from binance CSV file. Binance operation: {data["Operation"]}',
         )
-        importer.add_ledger_action(cursor, ledger_action)
+        importer.add_ledger_action(write_cursor, ledger_action)
+
+
+class BiananceEarnProgram(BinanceSingleEntry):
+    AVAILABLE_OPERATIONS = (
+        'Simple Earn Flexible Interest',
+        'Simple Earn Flexible Subscription',
+        'Simple Earn Locked Rewards',
+        'Simple Earn Locked Redemption',
+        'Simple Earn Flexible Redemption',
+        'BNB Vault Rewards',
+        'Swap Farming Rewards',
+    )
+
+    def process_entry(
+            self,
+            write_cursor: DBCursor,
+            importer: BaseExchangeImporter,
+            timestamp: Timestamp,
+            data: BinanceCsvRow,
+    ) -> None:
+        asset = data['Coin']
```

Comment:
What about all possible errors this function can raise? Where are they caught? Why don't we write them in the docstring? Why is there no docstring? Similar for all import files actually ...